### DISCUSSION TITLE
Migrate from LAUNCHXL-CC1352P-2 to Slaesh's CC2652RB

### DISCUSSION THREAD

**Discussion by zbuh** [Koenkk/zigbee2mqtt#8394] [General]
Is it possible to migrate the network from LAUNCHXL-CC1352P-2 to Slaesh's CC2652RB without repairing?

Thanks

**Comment by Koenkk** [comment]
Yes, see https://www.zigbee2mqtt.io/information/FAQ.html#what-does-and-does-not-require-repairing-of-all-devices

**Reply by zbuh** [reply]
Thanks, missed that.
